.github/workflows/publish-docker.yml

```diff
@@ -24,31 +24,23 @@ jobs:
     runs-on: ubuntu-latest
     needs: tests
     steps:
-    - uses: actions/checkout@v4
-    - name: Build the Docker image
-      run: docker build . -t weaver
-    - name: Push the Docker image
-      if: github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags/')
```

Comment:
You remove the condition to run this only when merged to main or when tags are added, is this intended?

Reply:
for debugging only.  The login isn't working, but I figured out why the previous solution wasn't working.

I'm not sure why the SAME secrets won't work with the login action.